max_obstacle_height: 0.60  # assume something like an arm is mounted on top of the robot
obstacle_range: 11.9
raytrace_range: 11.9

inflation_radius: 0.80

footprint: [[-0.4, -0.4], [-0.4, 0.4], [0.4, 0.4],[0.4, -0.4]]


